types/types.go

```diff
@@ -188,6 +189,13 @@ type Frontend struct {
 	Redirect             *Redirect             `json:"redirect,omitempty"`
 }
 
+// Hash returns the hash value of a Frontend struct.
+func (f *Frontend) Hash() string {
+	hash, _ := hashstructure.Hash(f, nil)
```

Comment:
Should handle the `error` here. If the hash goes sideways, you are going to want to log or potentially return an error.

Reply:
Agree. I think I took a shortcut because from te definition of the `Hash()` it is not able to produce an error with the current definition of the `Frontend` struct. But that's not very kind on the people doing changes in the future.

I'll create fixes for this (today or tomorrow probably).

Reply:
Implemented in d44fde676f86f2d039b2b8e4798f6b25a5d98e3a.